packages/hasura-storage-js/src/hasura-storage-api.ts

```diff
@@ -1,5 +1,6 @@
 import fetch from 'cross-fetch'
 
+import { fetchUpload } from './utils'
```

Comment:
Why are there changes to hasura-storage-api in this PR?

Reply:
`toIso88591` was not exposed for tests. I reverted.